### PR TITLE
[DIA-2210] Create a regression test for DIA-2106

### PR DESCRIPTION
**Pull request by bohdan-go-wombat**
# Task
Jira - https://sourcepoint.atlassian.net/browse/DIA-2210

# Description
This PR introduces regression tests that cover the exposure of UUID of CCPA and GDPR via CCPAConsent and GDPRConsent accordingly. Those tests will fail if we decide to protect UUID.

### CODE REVIEW

**Review by carmelo-iriti — APPROVED** [review]

**Review by andresilveirah — APPROVED** [review]

**Comment by andresilveirah** [comment]
I think we should merge this PR even if the UI test is failing. (this PR has no impact on UI tests)

**Comment by carmelo-iriti** [comment]
**Don't merge directly**, 
this code need to be rebased on the development branch because we have a new release

**Comment by bohdan-go-wombat** [comment]
@andresilveirah I think that makes sense, when other checks, except the UI one pass - I will merge it

@carmelo-iriti the branch has been rebased onto `develop`

**Pull request merged**
